ts/features/fims/history/saga/handleGetFimsHistorySaga.ts

```diff
@@ -29,9 +30,12 @@ export function* handleGetFimsHistorySaga(
       extractFimsHistoryResponseAction,
       getHistoryResult
     );
+    trackFailureIfNeeded(resultAction);
     yield* put(resultAction);
   } catch (e) {
-    yield* put(fimsHistoryGet.failure((e as Error).toString()));
+    const reason = `${e}`;
```

Comment:
```suggestion
    const reason = JSON.stringify(e);
```
what do you think? this way we should avoid an error message along the lines of `[object object`

Reply:
Done it